frameworks/hdfs/integration/tests/test_upgrade.py

```diff
@@ -24,10 +24,6 @@ def setup_module(module):
     uninstall()
 
 
-def teardown_module(module):
```

Comment:
I want to go look at the HDFS cluster as it's running in a failing test scenario.